frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -1,48 +1,54 @@
 <template lang='pug'>
-//- TODO: move this into modal
-main.main#create-group-page
-  .steps
-    router-link.step(
+modal-base-template
+  .steps(v-if='currentStep + 1 < config.steps.length')
+    button.step(
       v-for='(step, index) in config.steps'
       :key='index'
-      :to='{name: step}'
       :class='[currentStep === index ? "active" : "", currentStep < index ? "next" : ""]'
+      @click='redirect(step)'
+      v-if='step != "GroupWelcome"'
     ) {{ index + 1 }}
 
+  .wrapper.mobile-steps.subtitle
+    i18n(tag='span') Step
+    |  {{ currentStep }}
+    i18n(tag='span')  of
+    |  {{ config.steps.length + 1 }}
+
   transition(name='fade' mode='out-in')
-    router-view(
+    component(
+      :is='content'
       :group='form'
       :v='$v.form'
       @next='next'
       @focusref='focusRef'
       @input='payload => updateGroupData(payload)'
     )
+      .buttons(v-if='currentStep + 1 < config.steps.length')
+        button.is-outlined(
+          @click='prev'
+          data-test='prevBtn'
+        )
+          i18n {{ currentStep === 0 ? 'Cancel' : 'Back' }}
```

Comment:
Improper i18n usage (never use Vue.js variable syntax)

Reply:
What do you recommend?